lib/lapis_lazuli/generators/cucumber/template/config/cucumber.yml

```diff
@@ -1,11 +1,10 @@
 ################################################################################
-# Copyright <%= config[:year] %> spriteCloud B.V. All rights reserved.
-# Generated by LapisLazuli, version <%= config[:lapis_lazuli][:version] %>
-# Author: "<%= config[:user] %>" <<%= config[:email] %>>
+# Copyright 2017 spriteCloud B.V. All rights reserved.
```

Comment:
Should be 2018, but Wasnt  <%= config[:year] %> a good setup?

Reply:
Accidentally overwritten this. I've put the code back.